runtime/src/comm/ugni/comm-ugni.c

```diff
@@ -5011,6 +5011,7 @@ void consume_all_outstanding_cq_events(int cdi)
         //
         atomic_store_bool((atomic_bool*) (intptr_t) post_desc->post_id, true);
       }
+      CQ_CNT_DEC(cd);
```

Comment:
That's funny -- I think that for the AM handler's permanently bound comm domain we've been incrementing the CQ count as we initiated transactions but never decrementing it as the CQ events arrived and we consumed them.  But it didn't matter, because since the CD was permanently bound we never looked at the outstanding CQ count when "allocating" it.

Reply:
Right, we discovered that a while ago, but came to the same conclusion. Meant to note that in the PR message, but forgot to.

Reply:
It's amazing how fast I can forget stuff these days!   :-)